quickjs.c

```diff
@@ -6951,6 +7059,24 @@ JSValue JS_GetPropertyInternal(JSContext *ctx, JSValueConst obj,
     }
 }
 
+force_inline JSValue JS_GetPropertyInternalWithIC(JSContext *ctx, JSValueConst obj,
+                                                  JSAtom prop, JSValueConst this_obj,
+                                                  InlineCache *ic, int32_t offset, 
+                                                  BOOL throw_ref_error) 
+{
+    uint32_t tag;
+    JSObject *p;
+    tag = JS_VALUE_GET_TAG(obj);
+    if (unlikely(tag != JS_TAG_OBJECT))
+        goto slow_path;
+    p = JS_VALUE_GET_OBJ(obj);
+    offset = get_ic_prop_offset(ic, offset, p->shape);
+    if (likely(offset >= 0))
+        return JS_DupValue(ctx, p->prop[offset].u.value);
+    slow_path:
+        return JS_GetPropertyInternal(ctx, obj, prop, this_obj, ic, throw_ref_error);      
```

Comment:
```suggestion
slow_path:
    return JS_GetPropertyInternal(ctx, obj, prop, this_obj, ic, throw_ref_error);
```
Aside: can you make sure there's no trailing whitespace anywhere? LMK if you need a perl one-liner to fix that.